lms/templates/admin/instance.html.jinja2

```diff
@@ -50,43 +31,69 @@
+    <fieldset class="box mt-6">
+        {{ macros.registration_preview(request, instance.lti_registration) }}
+    </fieldset>
+
+    <fieldset class="box">
+        {{ macros.disabled_text_field("Consumer key", instance.consumer_key) }}
+        {{ macros.disabled_text_field("Deployment ID", instance.deployment_id) }}
+        {{ macros.disabled_text_field("LMS URL", instance.lms_url) }}
+    </fieldset>
 
-    {{ read_only_field("Tool consumer instance GUID", instance.tool_consumer_instance_guid) }}
-    {{ read_only_field("Tool consumer info product family code", instance.tool_consumer_info_product_family_code) }}
-    {{ read_only_field("Tool consumer instance description", instance.tool_consumer_instance_description) }}
-    {{ read_only_field("Tool consumer instance URL", instance.tool_consumer_instance_url) }}
-    {{ read_only_field("Tool consumer instance name", instance.tool_consumer_instance_name) }}
-    {{ read_only_field("Tool consumer instance contact email", instance.tool_consumer_instance_contact_email) }}
-    {{ read_only_field("Tool consumer info version", instance.tool_consumer_info_version) }}
+    <fieldset class="box">
+        <legend class="label has-text-centered">Tool Consumer</legend>
+        {{ macros.disabled_text_field("GUID", instance.tool_consumer_instance_guid) }}
+        <div class="columns">
+            <div class="column is-half">
+                {{ macros.disabled_text_field("Family code", instance.tool_consumer_info_product_family_code) }}
+            </div>
+            <div class="column is-half">
+                {{ macros.disabled_text_field("Description", instance.tool_consumer_instance_description) }}
+            </div>
+        </div>
+        <div class="columns">
+            <div class="column is-half">
+                {{ macros.disabled_text_field("URL", instance.tool_consumer_instance_url) }}
+            </div>
+            <div class="column is-half">
+                {{ macros.disabled_text_field("Name", instance.tool_consumer_instance_name) }}
+            </div>
+        </div>
+        <div class="columns">
+            <div class="column is-half">
+                {{ macros.disabled_text_field("Email", instance.tool_consumer_instance_contact_email) }}
+            </div>
+            <div class="column is-half">
+                {{ macros.disabled_text_field("Version", instance.tool_consumer_info_version) }}
+            </div>
+        </div>
 
-    {{ read_only_field("Custom Canvas API domain", instance.custom_canvas_api_domain) }}
+    </fieldset>
 
-    <fieldset class="box mt-6">
+    <fieldset class="box">
         <legend class="label has-text-centered">Canvas settings</legend>
+        {{ macros.disabled_text_field("Custom Canvas API domain", instance.custom_canvas_api_domain) }}
+        {{ macros.disabled_text_field("Developer key", instance.developer_key) }}
         {{ settings_checkbox("Canvas sections enabled", "canvas", "sections_enabled") }}
```

Comment:
I haven't been able to set the settings side by side. I'll have another go in a different PR. I won't be defeated by CSS.